example/cumulus-tf/main.tf

```diff
@@ -123,11 +123,13 @@ module "cumulus" {
 
   sts_credentials_lambda_function_arn = data.aws_lambda_function.sts_credentials.arn
 
-  archive_api_port            = var.archive_api_port
-  private_archive_api_gateway = var.private_archive_api_gateway
-  api_gateway_stage = var.api_gateway_stage
+  archive_api_port              = var.archive_api_port
+  private_archive_api_gateway   = var.private_archive_api_gateway
+  api_gateway_stage             = var.api_gateway_stage
   log_api_gateway_to_cloudwatch = var.log_api_gateway_to_cloudwatch
-  log_destination_arn = var.log_destination_arn
+  log_destination_arn           = var.log_destination_arn
+  ingest_logs_to_elk            = var.ingest_logs_to_elk
```

Comment:
Seems like you wouldn't need both of these variables.  If log_destination_arn was set, I'd assume we want to log to it.  Maybe that's not a good assumption?

Reply:
I went back and forth on this but yeah it's not really needed. It now just checks for a destination and if it's not set uses log2elasticsearch